pkgs/tools/graphics/vulkan-tools/default.nix

```diff
@@ -1,22 +1,31 @@
-{ stdenv, fetchFromGitHub, cmake, python3, vulkan-loader, vulkan-headers,
-  glslang, pkgconfig, xlibsWrapper, libxcb, libXrandr, wayland }:
+{ stdenv, lib, fetchFromGitHub, cmake, python3, vulkan-loader,
+ vulkan-headers, glslang, pkgconfig, xlibsWrapper, libxcb,
+ libXrandr, wayland }:
 
 stdenv.mkDerivation rec {
   pname = "vulkan-tools";
-  version = "1.2.131.1";
+  version = "1.2.141.0";
 
   src = fetchFromGitHub {
     owner = "KhronosGroup";
     repo = "Vulkan-Tools";
     rev = "sdk-${version}";
-    sha256 = "0ws47ansrr8cq4qjf6k4q0ygm9wwd3w7mhwqcl1qxms8lh5vmhfq";
+    sha256 = "1ch56ihm7rmilipfyc4i4ww7l6i20fb3qikkpm1ch43kzn42zjaw";
   };
 
   nativeBuildInputs = [ cmake pkgconfig ];
   buildInputs = [ python3 vulkan-headers vulkan-loader xlibsWrapper libxcb libXrandr wayland ];
   enableParallelBuilding = true;
 
-  cmakeFlags = [ "-DBUILD_ICD=OFF" "-DGLSLANG_INSTALL_DIR=${glslang}" ];
+  libraryPath = lib.strings.makeLibraryPath [ vulkan-loader ];
+
+  dontPatchELF = true;
+
+  cmakeFlags = [
+    "-DBUILD_ICD=OFF"
```

Comment:
I know this existed before your patch, but still, could you add a comment explaining this please?

Reply:
Done